index.js

```diff
@@ -103,8 +103,8 @@ function compile(source, options) {
     }];
 
     optionsCopy.presets = presets;
-    optionsCopy.sourceMaps = true;
-    if (result.map) {
+    optionsCopy.sourceMaps = options.sourceMap !== false && options.sourceMaps !== false;
```

Comment:
I don’t understand this logic @sebakerckhof - it looks as if you need to set BOTH sourceMap and sourceMaps to  a value !== false to get sourceMaps now. That seems weird.

Reply:
Remember that implicit undefined will also evaluate to true here.